schema/dmt/operations.go

```diff
@@ -0,0 +1,27 @@
+package schemadmt
+
+import (
+	"github.com/ipld/go-ipld-prime/datamodel"
+	"github.com/ipld/go-ipld-prime/node/bindnode"
+)
+
+// ConcatenateSchemas returns a new schema DMT object containing the
+// type declarations from both.
+//
+// As is usual for DMT form data, there is no check about the validity
+// of the result yet; you'll need to apply `Compile` on the produced value
+// to produce a usable compiled typesystem or to become certain that
+// all references in the DMT are satisfied, etc.
+func ConcatenateSchemas(a, b *Schema) *Schema {
```

Comment:
would be great to have a basic test/example so that there's an ongoing validation that concatenation of two basic schemas works

Reply:
In general I'm pro-testing but in this case I don't know what would provide useful coverage in exchange for the additional volume of test code.  We really are just essentially conjoining two maps here.

If there were more interesting and complex logical operations here, I would certainly want to test it, but what this does is so trivial I don't know if it's worth it.